Fluent/Simulation.ipynb

```diff
@@ -6,43 +6,43 @@
     "\n",
     "> **_In simple words: Current tutorial teaches users to fish, rather than giving them a fish._**\n",
     "\n",
-
-  
+    "# General Recommended Structure for Writing Domain File\n",
+    "\n",
     "\n",
     "## Setting up a CFD simulation has following four steps:\n",
     "\n",
-    "**1. ###### Setup Model/s:**   \n",
+    "1. ###### Setup Model/s:   \n",
     "According to the physics of the flow field user will select required model/s to simulate the flow.\n",
     "\n",
-    "**2. ###### Setup Working Fluid/s & Solid/s:**   \n",
+    "2. ###### Setup Working Fluid/s & Solid/s:   \n",
     "User will define the physical and thermodynamical properties of the working fluid/s and solid/s in the problem.    \n",
     "\n",
-    "**3. ###### Setup Boundary & Zone Conditions:**    \n",
+    "3. ###### Setup Boundary & Zone Conditions:    \n",
     "Solving the governing equations of the flow (i.e. system of partial differential equations) requires well-defined boundary conditions within the CFD domain. These conditions are selected and defined in this step.\n",
     "\n",
-    "**4. ###### Setup Solution Methods:**    \n",
+    "4. ###### Setup Solution Methods:    \n",
     "In CFD simulations the governing equations of the flow are solve numerically. Based on the physics of the problem appropriate numerical schemes and solution methods are selected at this step.\n",
     "\n",
-    "In the following section the details for the above four steps for the CFD simulation setup for **2D Laminar Unsteady Flow Over a Cylinder** are explained in great details. It should be noted that the path for defining conditions and other settings are provided in `command line` format. Users can access exact same settings and options by following the provided path via the tree of progress or pull down menu in ANSYS FLUENT. The summary of the steps to take for CFD simulation setup for problem of 2D laminar unsteady flow over a cylinder are as follows:\n",
+    "In the following section the details for the above four steps for the CFD simulation setup for <font color='red'>**insert the topic of the problem**</font> are explained in great details. It should be noted that the path for defining conditions and other settings are provided in `command line` format. Users can access exact same settings and options by following the provided path via the tree of progress or pull down menu in ANSYS FLUENT. The summary of the steps to take for CFD simulation setup for problem of 2D laminar flow over a flat plate are as follows:\n",
     "\n",
     "<br>\n",
     "\n",
-      
+    "<font color='red'>Insert the equivalent summary of commands for your problem of interest similar to the following sample. THESE COMMANDS SHOULD BE ADAPTED (i.e. ADDED OR REMOVED) BASED ON YOUR CASE STUDY. DO NOT COPY/PASTE THE SAME COMMANDS as the SAMPLE if not applicable (delete this paragraph when read)</font>  \n",
     "\n",
```

Comment:
The way that the command line paths are corrected is based on the FLUENT's tree of progress, not the command line accessible using FLUENT console. Choices for tree of progress are visible to users and subjected to potential changes in future release, but the path of command line are rarely possible to be changes. Thus, please revert the changes to the original format and only change/add the additional settings or boundary conditions.